### PR TITLE
chore(core): maintenance mode works again for non-logged in users

### LINKED ISSUE #8446: resources/maintenance view is missing
The file was accidentally forgotten from #8372

### PR DESCRIPTION
**Pull request by juho-jaakkola**
Adds the missing view that prints the maintenance mode page for non-logged in users.

Refs #8372
Fixes #8446

### CODE REVIEW

File: views/default/resources/maintenance.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+$message = $site->getPrivateSetting('elgg_maintenance_message');
+
+if (!$message) {
+	$message = elgg_echo('admin:maintenance_mode:default_message');
+}
+
+elgg_load_css('maintenance');
+
+header("HTTP/1.1 503 Service Unavailable");
```
Comment (mrclay) [inline views/default/resources/maintenance.php]: Was this in the original? Just curious, where was this code before?

Comment (juho-jaakkola) [inline views/default/resources/maintenance.php]: Yes, original: https://github.com/Elgg/Elgg/pull/8372/files#diff-9e9d8b4023c1a5a1f8608d6f3456306dL751

**Comment by mrclay** [comment]
LGTM, merge when build passes

File: views/default/resources/maintenance.php (hunk 1 above)
Comment (juho-jaakkola) [inline views/default/resources/maintenance.php]: Added here https://github.com/Elgg/Elgg/pull/5783 (third commit)

**Pull request merged**